.github/pull_request_template.md

```diff
@@ -0,0 +1,35 @@
+# Changes
```

Comment:
Does this not inherit pr templates from here? https://github.com/orgs/Health-Informatics-UoN/discussions/2

If so would rather keep those defaults for consistency